frontend/src/components/employees/index.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import Table from '@material-ui/core/Table';
+import TableHead from '@material-ui/core/TableHead';
+import TableBody from '@material-ui/core/TableBody';
+import TableRow from '@material-ui/core/TableRow';
+import TableCell from '@material-ui/core/TableCell';
```

Comment:
These imports can be changed to` import {TableCell, TableRow, TableBody, TableHead, Table, Button, Box} from '@material-ui/core';` you can still have each element on a new line. What I am suggesting is what I am used to, not sure if it is the accepted

Reply:
What I'm worried about is that this would import the entire `@material-ui/core` into the bundle, including elements we don't need. I'm not sure how clever webpack is about this, but this is how examples on material ui website were structured.

Reply:
Ah, fair eniough

Reply:
https://material-ui.com/guides/minimizing-bundle-size/

This explains why this is happening, and your way of doing it is a solution. I think it will only happen during the development build.

Option 2 provides a solution using babel plugins which will keep the bundle size small on developer build and allow the nice new imports.